.gitignore

```diff
@@ -127,3 +127,13 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+
+# Database
+*.db
+*/*.db
```

Comment:
Does this work to encapsulate both L133-134?
```
**/.db
```

Reply:
no

Reply:
You might need to rm git cache